test/util.py

```diff
@@ -14,13 +14,19 @@
 class ReturnCodeAssertionError(AssertionError):
     """Error to be raised when the return code of an executed process is not as expected."""
 
-def run_devnet_in_background(sleep_seconds=0):
-    """Run starknet-devnet in background. Return the process handle. Optionally sleep."""
-    command = ["poetry", "run", "starknet-devnet", "--host", HOST, "--port", PORT]
+def run_devnet_in_background(*args, sleep_seconds=3):
+    """
+    Runs starknet-devnet in background.
+    By default sleeps 1 second after spawning devnet.
```

Comment:
Isn't the default 3 seconds now?

Reply:
Right, I was experimenting with 1 second. Will change.

Reply:
Changed: https://github.com/Shard-Labs/starknet-devnet/pull/40/commits/17c17ee77c45d854787d053bace2f200de2bb31b